whois-nrtm4/src/main/java/net/ripe/db/nrtm4/dao/NrtmKeyConfigDao.java

```diff
@@ -23,29 +25,51 @@ public class NrtmKeyConfigDao {
         this.writeTemplate = new JdbcTemplate(writeDataSource);
     }
 
-    public byte[] getPrivateKey() {
-        return readTemplate.queryForObject("SELECT private_key FROM key_pair", (rs, rowNum) -> rs.getBytes(1));
+    public byte[] getActivePrivateKey() {
```

Comment:
Do we need to check if getActivePrivateKey and getActivePublicKey return just one row??
I mean, we shouldn't have 2 active keys but in case that we have a bug and there are two active keys in database...we maybe have a problem here?

Reply:
Yes in that case it will throw an exception